catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/SubscriptionsPersistentStoreImpl.java

```diff
@@ -46,6 +47,10 @@
 
   private static final String ID = "id";
 
+  @VisibleForTesting static final int START_INDEX = 0;
+
+  @VisibleForTesting static final int PAGE_SIZE = 1000;
```

Comment:
👍  Definitely prefer this over importing constants from _persistence-core-impl_.